test/blockstore-test.js

```diff
@@ -1,13 +1,3 @@
-/**
- * test/blockstore-test.js - blockstore test for hsd
- * Copyright (c) 2019, Braydon Fuller (MIT License).
- * Copyright (c) 2019, Mark Tyneway (MIT License).
```

Comment:
There's a few copyright lines like this that got deleted also. Not sure what the rules are about that...

Reply:
These are tests, I don't think there's anything copyright worthy here ))))

Reply:
It could be and I can revert ofc, but decided to get rid of them from tests, as I don't think they serve much purpose in these tests.